compiler/plugins/target/AMD-AIE/iree-amd-aie/Target/mm_npu4_peano.cc

```diff
@@ -25,6 +22,15 @@ void matmul_vectorized_i8_i32(const int8 * __restrict pA, unsigned offsetA, cons
   const unsigned size_B = L0_K * L0_N;
   const unsigned size_C = L0_M * L0_N;
 
+  v64int8 A0;
```

Comment:
I have seen issues with loop pipelining if these are defined in the inner loops.